src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    version: version,
+    whells: wheels,
```

Comment:
if variables and keys have the same names, you can write them like this

```suggestion
  const robot = {
    name,
    version,
    wheels,
```